redirects.txt

```diff
@@ -1,7 +1,14 @@
-"porting/introduction.rst" "porting/Intro.rst"
```

Comment:
this doesn't look right. I'm pretty sure: 

the left side is the old url and someone might go to. Probably we linked it from some external page, people created bookmarks, etc. for all intents and purposes this will never change

the right side is the new/current location of content that we want to redirect to.

so 

"porting/introduction.rst" "porting/introduction/index.rst"